server/src/test/java/org/elasticsearch/search/DefaultSearchContextTests.java

```diff
@@ -178,6 +182,20 @@ public void testPreProcess() throws Exception {
             ParsedQuery parsedQuery = ParsedQuery.parsedMatchAllQuery();
             context3.sliceBuilder(null).parsedQuery(parsedQuery).preProcess(false);
             assertEquals(context3.query(), context3.buildFilteredQuery(parsedQuery.query()));
+
+            when(queryShardContext.getIndexSettings()).thenReturn(indexSettings);
+            when(indexService.newQueryShardContext(anyInt(),anyObject(),anyObject(),anyString())).thenReturn(queryShardContext);
```

Comment:
Looks like, line 187 in not necessary

Reply:
Thanks for review , it is not necessary. I think i added in my first attempt to include use-case with routings in test. I've reformatted code with intelliJ   - sorry for formatting issues.